### PR TITLE
fix: display openssf best practices badge

### PR DESCRIPTION
**Pull request by degenaro**
## Types of changes

- [ ] Hot fix (emergency fix and release)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation (change which affects the documentation site)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Release (`develop` -> `main`)

## Quality assurance (all should be covered).

- [ ] My code follows the code style of this project.
- [x] Documentation for my change is up to date?
- [ ] My PR meets testing requirements.
- [ ] All new and existing tests passed.
- [x] All commits are signed-off.

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

# Before you merge

- Ensure it is a 'squash commit' if not a release.
- Ensure CI is currently passing
- Check sonar. If you are working for a fork a maintainer will reach out, if required.

### CODE REVIEW

**Comment by degenaro** [comment]
```
Congratulations!
According to the information that you've provided,
your project has earned the OpenSSF best practices 'passing' badge.
For more information, see this URL:
https://www.bestpractices.dev/en/projects/9408

Please add your badge to your project and/or repository site;
you've earned it:
* In markdown you can add this:
  [![OpenSSF Best Practices]([https://www.bestpractices.dev/projects/9408/badge)](https://www.bestpractices.dev/projects/9408](https://www.bestpractices.dev/projects/9408/badge)%5D(https://www.bestpractices.dev/projects/9408))
* In HTML you can add this:
  <a href="https://www.bestpractices.dev/projects/9408"><img src="https:///projects/9408/badge"></a>

We strongly encourage you to add or improve justification text,
including adding URLs that point to more information.
If you have suggestions on how to improve the criteria or
badge web application, please file an issue at:
https://github.com/coreinfrastructure/best-practices-badge/issues
One of our goals is to encourage projects to make some improvements.
We put some of these reports on the public wiki; please let us know
if you do NOT want the changes publicly posted at
https://github.com/coreinfrastructure/best-practices-badge/wiki/Impacts
Thank you, and congratulations.

--- David A. Wheeler, OpenSSF Best Practices Badge Lead
```

File: README.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ Consult [contributors](https://github.com/oscal-compass/compliance-trestle/graph
 
 ______________________________________________________________________
 
+<a href="https://www.bestpractices.dev/projects/9408"><img src="https://www.bestpractices.dev/projects/9408/badge"></a>
```
[the remark this answers is not in the document]
Comment (degenaro) [reply README.md]: Fixed. 

(and added CodeQL to reply on OpenSSF response)

**Review by jpower432 — APPROVED** [review]
LGTM

**Review by butler54 — CHANGES_REQUESTED** [review]
Please also add to `docs/index.md`

**Review by butler54 — APPROVED** [review]

**Pull request merged**